src/cephadm/cephadm

```diff
@@ -4542,8 +4542,8 @@ def command_ceph_volume(ctx):
         privileged=True,
         volume_mounts=mounts,
     )
-    verbosity = CallVerbosity.VERBOSE if ctx.log_output else CallVerbosity.VERBOSE_ON_FAILURE
```

Comment:
Do we also want to remove this since we're no longer using the setting? 
https://github.com/ceph/ceph/blob/601eb5af5a4fe543fcf56767fd035838b1adc5ad/src/cephadm/cephadm#L7594-L7598

Reply:
I think that to have the complete output of the ceph-volume command (or any other command) is only going to help us to fill logs...  imho, the thing to have is if we have a real problem processing the information that comes from ceph-volume and what is the problem.
I searched for the parameter and didn't found.... i thought it was removed time ago. I vote for remove the parameter

Reply:
Removed!